concordia/static/js/contribute.js

```diff
@@ -415,3 +415,37 @@ $tagEditor
 
         displayMessage('error', message, 'tags-save-result');
     });
+
+var hideTooltipCallback = function() {
+    // wait a couple seconds and then hide the tooltip.
+    var hideTooltip = function(tooltipButton) {
+        return function() {
+            tooltipButton.tooltip('hide');
+        };
+    };
+    setTimeout(hideTooltip($(this)), 3000);
+};
+var $copyUrlButton = $('#copy-url-button');
+$copyUrlButton.tooltip();
+$copyUrlButton.on('click', function() {
+    var $currentAssetUrl = $('#currentAssetUrl');
+    $currentAssetUrl.removeClass('d-none');
+    var currentAssetUrl = document.getElementById('currentAssetUrl');
+    currentAssetUrl.select();
+    var tooltipMessage = '';
+    try {
+        document.execCommand('copy');
+        // Show the tooltip with a success message
+        tooltipMessage = 'This link has been copied to your clipboard';
+    } catch (e) {
+        // Display an error message in the tooltip
+        tooltipMessage = 'Could not access your clipboard';
```

Comment:
We should add `Raven.captureException(e)` here so we can collect metrics on how frequently this happens